src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -4380,6 +4406,14 @@ subroutine calcstress(p,c,x,bsun,bsha,gb_mol,gs_mol_sun,gs_mol_sha,qsatl,qaf, &
        if (soilflux<0._r8) soilflux = 0._r8
        qflx_tran_veg(p) = soilflux
     endif
+
+    !save predawn vegwp
+    g = patch%gridcell(p)
+    if (night .and. get_local_time(grc%londeg(g))<(isecspday/2)) then
+       vegwp_pd(p,:) = x
```

Comment:
This isn't your fault, but the "x" array is a bad generic name to use. This predates your work, but it would be better if if was vegwp_old or some better name. I'll let you decide if you are up to changing it or not.

It's also assigning the whole array of "x", so a nitpick is that it would be clearer that it's an array assignment if it was

`vegp_pd(p,:) = x(:)`

Reply:
It may predate this PR, but that is also my handiwork! I agree with your point, but suggest we table this for a PHS refactor, which I see as inevitable at this point.

Reply:
Resolve for now. @djk2120 will address later.